tests/e2e/e2e_helm_install_test.go

```diff
@@ -0,0 +1,36 @@
+package e2e
+
+import (
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+)
+
+var _ = Describe("Test osm control plane installation with Helm", func() {
```

Comment:
@michelleN Could you update the test to add it to one of the tiers we defined in #1902 (specifically tier 2)?
```suggestion
var _ = DescribeTier2("Test osm control plane installation with Helm", func() {
```